source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/CalificationsEvaluationDto.cs

```diff
@@ -35,7 +35,9 @@ public class CalificationsEvaluationDto
 
         public List<string> Evaluators { get; set; }
 
-        public static CalificationsEvaluationDto Create(EmployeeEvaluation evaluation, List<string> evaluators, string currentPosition, string seniority)
+        public bool IsCompanyEvaluationDone { get; set; }
```

Comment:
It's not a matter of name but of evaluation states and what those states are for. I would like to analyze better this process and come up with other alternatives.